code/game/gamemodes/cult/cult.dm

```diff
@@ -167,21 +166,11 @@
 	to_chat(cult_mob, "\red [pick("You remember something from the dark teachings of your master","You hear a dark voice on the wind","Black blood oozes into your vision and forms into symbols","You have a vision of a [pick("crow","raven","vulture","parrot")] it squawks","You catch a brief glimmer of the otherside")]... [wordexp]")

```

Comment:
While ideally this would use messages more relevant to each cult (burning embers instead of black blood, or a vision of a gravestone with your word on it, etc), that would involve making a list of possible messages for each cult and thus kinda a pain.